manifest/manifest.go

```diff
@@ -281,6 +295,21 @@ func (m *Module) setInputsToProto(pbModule *pbsubstreams.Module) error {
 			pbModule.Inputs = append(pbModule.Inputs, pbInput)
 			continue
 		}
+		if input.Params != "" {
+			if i != 0 {
+				return fmt.Errorf("input.params must be the first input")
+			}
+
+			pbInput := &pbsubstreams.Module_Input{
+				Input: &pbsubstreams.Module_Input_Params_{
+					Params: &pbsubstreams.Module_Input_Params{
+						Value: input.Params,
```

Comment:
I don't think this is the right value. We want the parameters from the top-level `Params` mapping in the _Manifest_. That means we'll do a loop on that Manifest:Params and dig into those nested structure to set the `Value` there.
The `input.Params` here would be equal to `"string"` always. But that's not the value we put here. Leave it empty, and loop thorough the top-level manifest's `params` map.